src/System.Device.Gpio/System/Device/Gpio/PinValue.cs

```diff
@@ -26,13 +26,42 @@ namespace System.Device.Gpio
         /// </summary>
         public static PinValue Low => new PinValue(0);
 
+        /// <summary>
+        /// Implicit conversion from int. 0 means low, everything else means high.
```

Comment:
nit: instead of saying "low" or "high" should we use `<see cref="PinValue.Low"` and similar for High?